### DISCUSSION TITLE
Hero 11 firmware update

### DISCUSSION THREAD

**Discussion by fxstein** [gopro/labs#122] [General]
Any news on when the new Hero 11 labs firmware will be ready? Need the communications fixes from H22.01.01.12.00 that was recently released.

Thanks!

**Comment by dnewman-gpsw** [comment]
We are not doing a Labs update for H222.01.01.12.00, as it was only a minor change, and Labs testing is time consuming. However there is another feature packed Labs release in the works, awaiting potential main firmware updates.
